package.json

```diff
@@ -25,7 +25,13 @@
     }
   },
   "scripts": {
-    "preinstall": "yarn --cwd ccd-definition-processor install",
+    "preinstall": "git submodule update --init --recursive && yarn --cwd ccd-definition-processor install",
```

Comment:
Updated `preinstall` to ensure project is cloned every we do `yarn install` so that the git submodule is checked out on Jenkins